test/Microsoft.AspNet.Mvc.DataAnnotations.Test/DataTypeClientModelValidatorProviderTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    public class DataTypeClientModelValidatorProviderTest
+    {
+        private readonly IModelMetadataProvider _metadataProvider = TestModelMetadataProvider.CreateDefaultProvider();
+
+        [Theory]
+        [InlineData(typeof(float))]
+        [InlineData(typeof(double))]
+        [InlineData(typeof(decimal))]
+        [InlineData(typeof(float?))]
+        [InlineData(typeof(double?))]
+        public void GetValidators_GetsNumericValidator_ForNumericType(Type modelType)
```

Comment:
Could we add another test for cases where it returns empty sequences for other data types?